src/identity/manager.rs

```diff
@@ -122,6 +123,17 @@ impl SecretManager<CaClient> {
     }
 }
 
+impl SecretManager<CustomSigner> {
+    pub fn origin(cfg: crate::config::Config) -> Result<Self, Error> {
+        let cache: HashMap<Identity, watch::Receiver<Option<tls::Certs>>> = Default::default();
+        let custom_signer = CustomSigner::new(cfg.custom_signer)?;
+        Ok(Self {
+            client: custom_signer,
+            cache: Arc::new(RwLock::new(cache)),
```

Comment:
I can't find any place in this PR that uses this cache.
Is it on just a place holder for something?

Reply:
https://github.com/istio/ztunnel/blob/4e5667f8ed0ef7ccd3cdfe8b6c67774fc0a62fbd/src/app.rs#L138
one of the examples, how we call fetch_certificate() - https://github.com/istio/ztunnel/blob/master/src/proxy/inbound.rs#L413